src/AccessibilityInsights.SharedUx/Utilities/HelperMethods.cs

```diff
@@ -50,6 +50,21 @@ public static Visibility GeneralFileBugVisibility
             }
         }
 
+        /// <summary>
+        /// Get a "File Issue" column width that is appropriate for the currently configured font size
+        /// </summary>
+        public static double FileIssueColumnWidth
+        {
+            get
+            {
+                switch (ConfigurationManager.GetDefaultInstance().AppConfig.FontSize)
```

Comment:
Do we want this method to include the 0 width option seen in `MainWindow`'s `BugColumnWidth` property? I think (but am not sure) it is a bug that `ScannerResultControl`'s `BugColumnWidth` is not currently 0 when there is no issue filing. This isn't a regression though, so this PR doesn't need to fix it.

Reply:
I was thinking that it made sense to keep the visibility and the size decoupled. Leaving it as is for now, but if/when it's a problem it could make sense to combine them.